tools/newRepo

```diff
@@ -19,56 +19,59 @@
 //  VP   V8P Y88888P  `8b8' `8d8'  88   YD Y88888P 88       `Y88P'
 
 const inquirer = require("inquirer");
-const client = require("../src/client.js"); 
+const client = require("../src/client.js");
```

Comment:
I'm not sure how did that pass the tests. [`xo`'s rules](https://github.com/sindresorhus/eslint-config-xo/blob/master/index.js) have `no-trailing-spaces` set to `error`.